instruments/qubitekk/cc1.py

```diff
@@ -54,6 +55,15 @@ def __init__(self, filelike):
             self._set_fmt = ":{} {}"
             self.TriggerMode = self._TriggerModeOld
 
+    def _ack_expected(self, msg=""):
+        if self._ack_on:
```

Comment:
This can be condensed down to:
```python
if self._ack_on and self.firmware[0] >= 2 and self.firmware[1] > 1:
    return msg
else:
    return None
```